apps/services/auth/ids-api/infra/identity-server.ts

```diff
@@ -71,8 +71,12 @@ export const serviceSetup = (services: {
       PersistenceSettings__BaseAddress: ref(
         (h) => `http://${h.svc(services.authIdsApi)}`,
       ),
-      PersistenceSettings__UserProfileBaseAddress:
-        'http://web-service-portal-api.service-portal.svc.cluster.local',
+      PersistenceSettings__SessionsBaseAddress: {
+        dev: 'http://web-services_sessions.services_sessions.svc.cluster.local',
```

Comment:
is there a particular reason for using underscores in the URL?

Reply:
Not that I'm aware of, do you know anything about that @saevarma ?

Reply:
As I understood the internal ingress urls the pattern for dev and staging is  `http://web-<service name>.<namespace>.svc.cluster.local` and the service name and namespace for the sessions service is set with underscore [here](https://github.com/island-is/island.is/blob/262fd4604f58c371bedbb2a75739242b7cacf469/apps/services/sessions/infra/sessions.ts#L3)

Reply:
@busla should we change the service name and namespace so that they don´t have underscores?

Reply:
Yes please change it to have a dash instead of an underscore for the `namespace`, `serviceName` and `workerName`. I should have spotted this during the original PR, sorry about that @valurefugl

Reply:
No prob, will fix, thanks.

Reply:
Fixed.